tests/utils/autoTester.js

```diff
@@ -358,20 +363,22 @@ module.exports.addStep = function (name, stepFunction) {
     doAddStep(name, stepFunction, false);
 }
 
+module.exports.add2sDelays = function (numDelays) {
```

Comment:
2 comments here:
1/ Instead of introducing a call to wait "2second delays" why don;t you just do a 
addWait(duration) {}
And the implementation in the doAddStep(....)
undrestand that you want to perform a wait timer of 'duration' seconds

2/ the name of that function, addWait() sounds better to me ?

Reply:
Well i guess there is a 3rd comment here related to the name, why 2s ? instead by specifying the duration as a parameter then that should work isn't it ?

Reply:
That is a cleaner, but more complicated solution - add the wait time as another value in the TestCase.
It is a bit more difficult actually, as the time would be part of the TestCase, but used in a strange way.  You would put the default 2s for all steps, and put the delay you want in the `addWait` steps, but with no function.
That would make the `autotester.js` script even harder to understand.
We had a delay in the `enableAuto` method, but that isn't used in recursive mode
And - the resolution of the delay isn't important, as the delays required are machine-dependent.  Maybe just call it `addEmptySteps()` ?